app/views/map/index.html.erb

```diff
@@ -0,0 +1,127 @@
+<div id='map' class="map container"> 
+</div>
+<div id="info" style="display: none;"></div>
+<label for="track">
+      track position
+      <input id="track" type="checkbox"/>
+    </label>
+<link href="https://unpkg.com/openlayers-indoorequal@latest/openlayers-indoorequal.css" rel="stylesheet" />
+<script src="https://unpkg.com/openlayers-indoorequal@latest/dist/openlayers-indoorequal.umd.min.js"></script>
+<script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.9.0/build/ol.js"></script>
+<script src="https://unpkg.com/elm-pep"></script>
+<style>
+    .map {
+        height: 1000px;
+        width: 1500px;
+    }
+    .level-control {
+        height: 400px;
+        width: 40px;
+    }
+</style>
+
+<script>
+
+const key = 'iek_VpPLmIcjE8vIQ7dVhFr3zw3lqTHj';
```

Comment:
I highly doubt that it's a good idea to include an api key as plain text in an HTML file.

Reply:
I think you could look at "Repository Secrets". Those are files that can store this kind of data but aren't public.